cluster/cluster_interface.ml

```diff
@@ -0,0 +1,214 @@
+(* Cluster interface *)
+
+open Idl
+
+let service_name = "cluster"
+let queue_name = Xcp_service.common_prefix ^ service_name
+let json_path = "/var/xapi/cluster.json"
+let xml_path = "/var/xapi/cluster"
+
+type debug_info = string
+[@@doc ["An uninterpreted string associated with the operation."]]
+[@@deriving rpcty]
+
+type cluster_name = string
+[@@doc ["Name of the cluster"]]
+[@@deriving rpcty]
+
+type address = IPv4 of string
+[@doc ["An IP address"]]
```

Comment:
Should this say "An IP address a.b.c.d" or similar to indicate that this is IPv4 and not a hostname or IPv6?